### ISSUE TITLE
[bitnami/mlflow] WARNING:  database "bitnami_mlflow" has a collation version mismatch

### ISSUE THREAD

**Issue by bhack** [bitnami/charts#29653]
### Name and Version

bitnami/mlflow 1.5.7

### What architecture are you using?

amd64

### What steps will reproduce the bug?

Upgrade to `mlflow-1.5.7` app version `2.16.2 `

### What is the expected behavior?

Just working as with other upgrades

### What do you see instead?

```
WARNING:  database "bitnami_mlflow" has a collation version mismatch
DETAIL:  The database was created using collation version 2.31, but the operating system provides version 2.36.
HINT:  Rebuild all objects in this database that use the default collation and run ALTER DATABASE bitnami_mlflow REFRESH COLLATION VERSION, or build PostgreSQL with the right library version.
```

**Comment by javsalgar** [comment]
Hi!

Is this only a warning or this is causing the installation to fail?

**Comment by bhack** [comment]
The installation doesn't fail but it seems slower and the postgres pod/container is fully spamming this messages.

**Comment by bhack** [comment]
See also:
https://www.postgresql.org/docs/current/sql-altercollation.html#SQL-ALTERCOLLATION-NOTES

**Comment by javsalgar** [comment]
From which version of the chart are you updating?

**Comment by bhack** [comment]
From the previous one.

**Comment by javsalgar** [comment]
You mean from chart version mlflow-1.5.6 to mlflow-1.5.7?

**Comment by bhack** [comment]
Yes but I followed all the updates over the time  so I cannot be 100% sure that it was exactly introduced in 1.5.6/1.5.7 update as I was not checking the postgres log on each upgrade.

**Comment by javsalgar** [comment]
Did you change any default values in the postgresql subchart?

**Comment by bhack** [comment]
No, It is vanilla.

**Comment by dgomezleon** [comment]
Hi @bhack ,

It seems that the warning could be fixed by executing:

```
REINDEX DATABASE bitnami_mlflow;
ALTER DATABASE bitnami_mlflow REFRESH COLLATION VERSION;
```

I'm going to create an internal task to implement this logic for upgrades.

**Comment by bhack** [comment]
Stale?

**Comment by juan131** [comment]
@bhack sorry for the long delay without responses, I just came across with this issue

Are you able to reproduce the issue if you set the parameter `tracking.runUpgradeDB` to `true` (see https://github.com/bitnami/charts/blob/main/bitnami/mlflow/values.yaml#L316-L318)? Please note this will add a init-container that performs an upgrade of the database before starting Mlflow main container.
